international-labour-organization.csl

```diff
@@ -110,14 +116,15 @@
   </macro>
   <macro name="pages">
     <group>
-      <label variable="page" form="short" suffix=" "/>
+      <label variable="page" form="short" plural="contextual" suffix=" "/>
       <text variable="page"/>
     </group>
   </macro>
   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="true" disambiguate-add-givenname="true">
     <layout prefix="(" suffix=")" delimiter="; ">
       <group delimiter=" ">
         <text macro="author-short"/>
+        <text variable="abstract"/>
```

Comment:
the abstract for in-text citations in a Harvard style? That has to be a mistake, no?